### DISCUSSION TITLE
Daily 13/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#55] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer segui haciendo correcciones en  la documentación y agregando contenido. También estuve revisando el  diagrama relacional y empecè a ver como serían el resto de los diagramas. No pude instalar Angular y solo llegué a compilar TS.
Hoy voy a seguir con la documentación y los diagramas. También con el contenido de la plataforma y trataré de instalar Angular. Por la tarde tengo clase sincrónica y a la noche nos juntamos con el equipo para terminar de acordar algunos temas.

**Comment by AhumadaCeleste** [comment]
Ayer cree isuue e historias de  usuario, también trabaje con el modelo relacional el cual seguí ajustando.  Hoy subí ambos al repositorio. Me llevo mucho tiempo pero de significativo aprendizaje.
Hoy tenemos Reunión de grupo y planeo arrancar con el contenido ya que estaba haciendo un curso de javaScript para tener algún a base de sintaxis, control de flujo, etc, antes de iniciar angular. No llegue a terminarlo pero espero poder avanzar con ello, que es contenido fuera de la cursada por fuera.

**Comment by Brian-Almada** [comment]
Ayer comencé a ver Angular muy por encima. Luego, estuve pensando en casos de usuarios pero los que se me ocurrían ya estaban.
Hoy voy a profundizar más en Angular.

**Comment by BraianTroncoso** [comment]
Ayer leí documentacion, hoy planeo seguir haciendo lo mismo.
La unica complejidad es el tiempo.

**Comment by KAAcevedo** [comment]
Asistí a mi primera clase de consultas del año (Comisión N°4, 18:30hs)
Estuve revisando un poco más exhaustivamente el Campus. Repasé inglés. Aún no instalé Angular, pero ya tengo Node y TypeScript. 
Tuvimos otro encuentro sincrónico y gracias a la puesta en escena ajustamos detalles de las tablas de la Base de datos.

**Comment by rpgrca** [comment]
Hoy estuve viendo más historias de usuario, ayudamos con el diagrama de base de datos, y mañana o pasado me pongo a hacer el diagrama de clases.
